core/src/main/java/com/google/instrumentation/trace/PropagationUtil.java

```diff
@@ -30,139 +28,76 @@
  * <li>version_id: 1-byte representing the version id.
  * <li>For version_id = 0:
  *     <ul>
- *     <li>version_format: &lt;trace-id&gt;&lt;span-id&gt;&lt;trace-options&gt;
- *     <li>trace-id: 16-byte array representing the trace_id.
- *     <li>span-id: 8-byte array representing the span_id.
- *     <li>trace-options: 4-byte array representing the trace_options. It is in little-endian order,
- *         if represented as an int.
+ *     <li>version_format: &lt;field&gt;&lt;field&gt;...
```

Comment:
The numbering in the Javadoc starts at 1, so you might need to use `<ol start="0">` or remove the numbering.